content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
+
+```shell
+kubectl get storageclass -n kubeflow
+
+NAME                   PROVISIONER                            AGE
+nfs (default)          cluster.local/nfs-client-provisioner   6h13m
+...
+```
+
+As you noticed we specified an optional parameter storageClass.defaultClass to true: 
+this will define *nfs* storage class as default.
+In this way during KubeFlow installation, all PVCs will be labelled
+with *nfs* storage class.
+
+## Finally: install KubeFlow!
+
+We finally prepared our on-prem multi-node Kubernetes cluster to manage transparently volumes
+using a NFS server: now you can finally install KubeFlow following the Vanilla on-prem KubeFlow installation guideline that you encountered in the top of this document.
+
+At the end of procedure you'll see that all PVCs defined by KubeFlow will be binded to some
+automatically created PV with the storage class *nfs*.
+
+Notice that the provisioner will create a directory for each PVC with a unique name related 
+to that PVC inside the root NFS directory.
+
+## In case of existing kubeflow installation
+
+In case you setup Dynamic Provisioning *after* the installation of KubeFlow, you must change the storage class on your existing unbinded PVCs.
+
+Unlikely, Kubernetes doesn't allow to change storage class of PVCs on the fly.
+
+To perform this task you need to:
+
+* download existing PVCs.
+* change in them storage class
+* delete and recreate them in the cluster.
+
+Download the three PVCs:
+
+```shell
+kubectl get pvc/mysql-pv-claim -n kubeflow -o yaml > mysql-pv-claim.yaml
+kubectl get pvc/minio-pvc -n kubeflow -o yaml > minio-pvc.yaml
+kubectl get pvc/katib-mysql -n kubeflow -o yaml > katib.yaml
+```
+
+And then modify files to add the right storageClassName under *spec* section:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pv-claim
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs
+  ...
+```
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: katib-mysql
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs-client
+  ...
+```
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: katib-mysql
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs-client
+  ...
+```
+
+Finally remove old ones:
+
+```shell
+kubectl delete -f mysql-pv-claim.yaml
+kubectl delete -f minio-pvc.yaml
+kubectl delete -f katib.yaml
+```
+
+and add them again:
+
+```shell
+kubectl apply -f mysql-pv-claim.yaml
+kubectl apply -f minio-pvc.yaml
+kubectl apply -f katib.yaml
+```
+
+As you'll see, the PVCs wil lbi binded to your NFS storage.
+
+## Limitations
+
+NFS is a remote filesystem that is high performant in reading while in writing it is slower.
+If you have to write a huge amount of data in your workflow, you will need to do some 
+consideration to understand if NFS filesystem is the right choice.
+
+It is preferable to use version 4 of NFS (instead 3): NFS 3 can have some problems like 
```

Comment:
* Change the start of the sentence to: "Ensure that you are using version 4 of NFS (instead of version 3):" - then remove the redundant sentence from the end of the paragraph
* `instead 3` --> `instead of version 3`
* `partial write` --> `partial writing`
* `and has no authentication` --> `and it does not support authentication`